### DISCUSSION TITLE
Latest update being flagged as Malware

### DISCUSSION THREAD

**Discussion by H4irBear** [valinet/ExplorerPatcher#2474] [General]
Just tried to install update 22621.2506.60 and had it flagged as malware.  Previous updates over the last year have been fine.  [Virustotal](https://www.virustotal.com/gui/file/80c19df6e5070b0c013d4ac7e7b4f2cca2d085b71e0b102aaf930a469089f8ef/detection) has 4 vendors showing it as Malware.

Any ideas as to why this is happening now?

**Comment by pyrates999** [comment]
false flag.  You would need to contact the antivirus vendors to tell them it is a false flag.

**Comment by Amrsatrio** [comment]
Because I've added a new DLL into the program, which contains the code for the properties window along with its texts/translations. May also be because I made the main DLL load that GUI DLL. Either ways it's a false positive. Webroot also flagged the first EP that fixes Win+Tab and more.

**Reply by H4irBear** [reply]
Thanks for the info. I have requested reclassification on Palo Alto Wildfire / Coretex which is what stopped my update.

**Reply by Amrsatrio** [reply]
I hope they know that it is open source and can check the source code of this, though 😅

**Comment by H4irBear** [comment]
They do, I linked to the GitHub in the request.

Thanks for the work improving the project!

**Comment by H4irBear** [comment]
Palo Alto have updated their verdict to benign.
